Lib/test/test_tempfile.py

```diff
@@ -1611,9 +1611,13 @@ def test_link_tmpfile(self):
                 tmp.flush()
                 fd = tmp.fileno()
 
-                os.link(f'/proc/self/fd/{fd}',
-                        filename,
-                        follow_symlinks=True)
+                try:
+                    os.link(f'/proc/self/fd/{fd}',
+                            filename,
+                            follow_symlinks=True)
+                except PermissionError as exc:
+                    # gh-136156: link() fails with PermissionError on Android
+                    self.skipTest(str(exc))
```

Comment:
Most of similar code in `test_os` and `test_posix` add more context, for example `f'os.link(): {exc!s}'`.